app/src/main/java/com/celzero/bravedns/util/Constants.kt

```diff
@@ -122,13 +124,14 @@ class Constants {
         const val PORT_VAL_UNKNOWN = "unknown"
 
         const val NOTIFICATION_ACTION = "NOTIFICATION_VALUE"
-        const val VPN_NOTIFICATION_ACTION = "STOP_RETHINKDNS"
+        const val STOP_VPN_NOTIFICATION_ACTION = "RETHINK_STOP"
+        const val DNS_VPN_NOTIFICATION_ACTION = "RETHINK_DNSONLY" // battery-saver dns-only
+        const val DNS_FIREWALL_VPN_NOTIFICATION_ACTION = "RETHINK_FULLMODE" // default dns+firewall
 
         const val NXDOMAIN = "NXDOMAIN"
 
-        const val WORK_MANAGER_TIMEOUT = 1*40*60*1000
+        val WORK_MANAGER_TIMEOUT = TimeUnit.MINUTES.toMillis(40) //1*40*60*1000
```

Comment:
Nit: There's no need for math in the comment. The code is more descriptive than the comment!

Reply:
Removed the comment.